nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-api/src/main/java/org/apache/nifi/web/controller/ControllerFacade.java

```diff
@@ -1489,6 +1492,12 @@ public int compare(AttributeDTO a1, AttributeDTO a2) {
             dto.setChildUuids(childUuids);
         }
 
+        // lineage duration
+        if (event.getLineageStartDate() > 0) {
```

Comment:
If we don't piggyback off of summarization, I believe this can be moved back.

Reply:
lineage duration was pulled out specifically because there was a case in which the duration was not properly populated. This was during early testing and may now be corrected by other changes. I will try to replicate.